src/libraries/Common/src/System/Net/Http/Http2/Hpack/StatusCodes.cs

```diff
@@ -1,4 +1,4 @@
-﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Copyright (c) .NET Foundation. All rights reserved.
```

Comment:
BOM change?

Reply:
Yeah. It depends on who's editor touched it last...